### ISSUE TITLE
Train DeepSpeech to obtain Librivox cClean test WER  below 10% with various training data sets

### ISSUE THREAD

**Issue by kdavis-mozilla** [mozilla/DeepSpeech#24]

**Comment by aaronzira** [comment]
Anyone have any updates on this? I was not able to break 39% WER with DS1 or DS2, but always stopped after I observed overfitting in the val loss (around 5-7 epochs for DS2). What sorts of numbers are others hitting? Any hyperparameter configs that look promising?
